src/multi_auth/providers/google.cr

```diff
@@ -61,7 +61,7 @@ class MultiAuth::Provider::Google < MultiAuth::Provider
 
   private def primary(field)
     primary = primary?(field)
-    raise "No primary in #{json[field]}" unless primary
+    raise "No primary in field #{field}" unless primary
```

Comment:
Just showing the field name that tried to get a primary. This should probably be moved to an official error like `MultiAuth::Provider::Google::InvalidPrimaryFieldError` or something... but I guess that could come later.